modules.d/90multipath/multipathd.service

```diff
@@ -1,9 +1,10 @@
 [Unit]
 Description=Device-Mapper Multipath Device Controller
 Before=iscsi.service iscsid.service lvm2-activation-early.service
-Wants=systemd-udev-trigger.service systemd-udev-settle.service local-fs-pre.target
-After=systemd-udev-trigger.service systemd-udev-settle.service
+Wants=local-fs-pre.target
 Before=local-fs-pre.target
+Wants=systemd-udevd-kernel.socket
+After=systemd-udevd-kernel.socket
```

Comment:
Explanation is only partially correct; it _had_ been necessary in the initramfs as events would be generated even before multipathd had been started, and the idea was to replay all events upon multipathd startup to ensure that we're not missing any configuration.
But this has been fixed now, and this patch is correct.